doc/conf.py

```diff
@@ -14,79 +14,79 @@
 import datetime
 import os
 
-project = 'azplugins'
+project = "azplugins"
 year = datetime.date.today().year
-copyright = f'2018-2020, Michael P. Howard. 2021-{year}, Auburn University.'  # noqa: A001
-author = 'Michael P. Howard'
-release = '1.0.0'
+copyright = f"2018-2020, Michael P. Howard. 2021-{year}, Auburn University."
+author = "Michael P. Howard"
+release = "1.0.0"
 
 # -- General configuration ---------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-default_role = 'any'
+default_role = "any"
 
-templates_path = ['_templates']
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+templates_path = ["_templates"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 extensions = [
-    'sphinx.ext.mathjax',
+    "sphinx.ext.mathjax",
 ]
 
-extensions += ['sphinx.ext.autodoc', 'sphinx.ext.autosummary']
+extensions += ["sphinx.ext.autodoc", "sphinx.ext.autosummary"]
 autodoc_docstring_signature = True
-autodoc_typehints_format = 'short'
-autodoc_mock_imports = ['hoomd.azplugins._azplugins']
-if os.getenv('READTHEDOCS'):
+autodoc_typehints_format = "short"
+autodoc_mock_imports = ["hoomd.azplugins._azplugins"]
+if os.getenv("READTHEDOCS"):
     autodoc_mock_imports += [
-        'hoomd._hoomd',
-        'hoomd.version_config',
-        'hoomd.md._md',
-        'hoomd.metal._metal',
-        'hoomd.mpcd._mpcd',
-        'hoomd.minimize._minimize',
-        'hoomd.hpmc._jit',
-        'hoomd.hpmc._hpmc',
+        "hoomd._hoomd",
+        "hoomd.version_config",
+        "hoomd.md._md",
+        "hoomd.metal._metal",
```

Comment:
Is there any part of azplugins that needs hoomd.metal? or hoomd.hpmc?

Reply:
This code is actually for building documentation, not testing. Due to HOOMD's conversion of plugins to be only internal (at least in their provided template), it is no longer possible to compile the documentation for azplugins without a HOOMD installation. Locally, this isn't a problem since you can do it after the build & install steps, but RTD doesn't want long-running builds. iirc, I failed at even installing HOOMD from conda-forge on RTD (can't remember why).

My solution was to modify the RTD build process:
https://github.com/mphowardlab/azplugins/blob/529ad29340c714bec1398b99e7ef652f94eed236/.readthedocs.yaml#L12-L15
so that it does a Python-only installation of both HOOMD and azplugins:
https://github.com/mphowardlab/azplugins/blob/529ad29340c714bec1398b99e7ef652f94eed236/doc/mock_install.py#L13-L25
This means that anything imported in either HOOMD or azplugins that is compiled needs to be mocked. That's why this list is only added when the RTD environment variable is set.

So, most of this list needs to stay, but I think HOOMD 5 did remove some compiled code (maybe `_jit` and `_metal`?). If yes, we should remove those because they are unused (but also don't cause any harm).

Reply:
How about we leave it as it is for now, and then revisit this as we change our azplugins documentation style to be consistent with hoomd?

Reply:
I pushed a commit to clean it up since I had the file open to look anyway :) So long as this passes, we will merge.